pedro-monteiro/1.i/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Sistemas de Informação</title>
+</head>
+<body>
+    <h3>Sistemas de Informação</h3>
```

Comment:
We should always start with `<h1>` before `<h2>` and `<h2>` before an `<h3>`. Please listen to this [podcast](http://hipsters.tech/acessibilidade-web-hipsters-21/) in order to understand why it is important to follow these rules!